stripe/src/main/java/com/stripe/android/model/Card.java

```diff
@@ -214,9 +286,18 @@ public Card(
                 null,
                 null,
                 null,
-                null);
+                null,
+                currency);
```

Comment:
Note that the `currency` field was already in the constructor... we just ignored it for some reason.